{{cookiecutter.project_slug}}/docker-compose.override.example.yml

```diff
@@ -46,8 +46,10 @@
 #     environment:
 #       <<: *x-environment
 #       SSH_AUTH_SOCK: /ssh-agent
+#       MEDIA_ROOT: /opt/media  # If you want to use a custom media path
```

Comment:
I don't see the use-case for moving MEDIA_ROOT to a different location inside the container. Is it worth putting in the example (which already contains many things) ?

Reply:
Oui c'est vrai :+1: